src/components/Book.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = ({ book }) => (
+  <tr>
```

Comment:
- **[OPTIONAL]** `book` prop are destructured from the function parameters in functional components to avoid additional variable declarations.